doc/release/RELEASE-PROTOCOL.md

```diff
@@ -131,16 +131,24 @@ Make sure that:
+1. To enhance communication within the team, the responsible person for the release must notify all stakeholders about the release two working days before its scheduled date and in 1-2 hours to prevent any actions on the environment on which the deployment is carried out.
+2. The responsible person have to take db snapshot manually via Amazon RDS in the `Snapshots` tab with name `env-db-date`.
+3. On the designated time and day, the responsible person runs the Release [Deploy] workflow for the sandbox and production environments from the release branch. They need to fill in the full release tag version (`X.Y.Z`) and choose the environment.
+4. After completing the triggered workflows, the responsible person must open the AWS console and verify that all tasks of the OpenSupplyHubStagingAppDD, OpenSupplyHubStagingApp, OpenSupplyHubProductionAppDD, and OpenSupplyHubProductionApp services in the ecsOpenSupplyHubStagingCluster and ecsOpenSupplyHubProductionCluster Amazon ECS clusters, respectively, have been restarted.
+In case workflow failed in the apply step responsible person have to run `Deploy to AWS` workflow for the sandbox and production environments from the release branch.
+5. The responsible person also needs to check that DedupeHub is up and running. To do this, they should open CloudWatch via the AWS console, navigate to the Log groups section in the menu, open logOpenSupplyHubProductionAppDD and logOpenSupplyHubStagingAppDD, then open the latest log stream of each log group. Ensure that there is a recent message: `INFO: Application startup complete.`
+6. Once the aforementioned steps are successfully completed, the person responsible for the release should also follow the instructions outlined in the Release section of the [RELEASE-NOTES.md](./RELEASE-NOTES.md) file for the released version. In case there is a need to run a command in the terminal of the Django container, follow [this instruction](https://opensupplyhub.atlassian.net/wiki/spaces/SD/pages/140443651/DevOps+Guidelines+for+Migration+Database+Snapshots+and+ECS+Management#All-the-steps-described-in-this-Document-should-be-run-by-DevOps-or-Tech-Lead-Engineers-only%5BhardBreak%5D%5BhardBreak%5D%5BhardBreak%5D%5BhardBreak%5D%5BhardBreak%5D%5BhardBreak%5D%5BhardBreak%5DHow-to-correctly-run-migrations-for-our-four-environments%3F---Even-if-it-will-be-done-in-the-OSDEV-564-JIRA-ticket%2C-we-need-to-have-instructions-for-the-current-state-of-the-infrastructure.).
+7. Notify the QA Engineer that the new version has been released, and they can commence testing the items listed under the *prod* column in [the QA Checklist sheet](https://docs.google.com/spreadsheets/d/
+8. Upon completing the release, the responsible person must notify stakeholders in the *#data_x_product* Slack channel that the releases to sandbox and production have concluded. Additionally, update the *Unreleased* version's status in Jira.
+1uinHJOPpGfrUNkewBxPVsDeDXnNx4dJnnX94LoBA_zk/edit?usp=sharing).
```

Comment:
What is this? Did you want to add a link?

Reply:
That was the link from the previous option. Fixed.